src/core/Group.h

```diff
@@ -68,6 +68,10 @@ class Group : public ModifiableObject
         QString defaultAutoTypeSequence;
         Group::TriState autoTypeEnabled;
         Group::TriState searchingEnabled;
+        Group::TriState browserIntegrationHideEntriesEnabled;
```

Comment:
These should be deleted since this is in custom data. Let the browser service worry about the name of the field.